rpm/soletta.spec

```diff
@@ -1,46 +1,57 @@
-%global soletta_major 0
-%global soletta_minor 0
-%global soletta_build 1
-%global soletta_tag beta13
+%global soletta_version 1
+%global soletta_current 1
+%global soletta_revision 0
+%global soletta_age 0
 
-%global soletta_duktape_version 1.2.2
-%global soletta_duktape_tag beta2
+%global soletta_major %(echo $(( %{soletta_current} - %{soletta_age} )))
```

Comment:
not sure if it's the way to do it on a spec file, but I assume you verified already